pages/api/performances/edit.js

```diff
@@ -14,29 +14,40 @@ export default async (req, res) => {
   const {
     id,
     name,
-    academicLevel,
     performers,
     choreographers,
     competitionLevel,
     danceSize,
     danceEntry,
     danceStyle,
     danceTitle,
+    performanceLink,
+    audioRecordingLink,
+    danceSizeID,
+    danceStyleID,
+    competitionLevelID,
     eventID,
     schoolID,
   } = req.body;
 
+  // TODO: patch edit per data field?
+  /** Optional FIELDS
+    danceSizeID,
```

Comment:
If you don't pass them in, does it update it to NULL?

Reply:
Aye let me try this out.